tools/importers/SynchronizeTypographyTokens.ts

```diff
@@ -0,0 +1,189 @@
+const axios = require('axios');
+const tokensFilesConfig = require('./configs/SynchronizeTypographyTokensConfig.json');
+const fileWriter = require('./helpers/fileWriter');
+const fileRead = require('./helpers/fileReader');
+const modifiers = require('./helpers/modifiers');
+const importerVariables = require('./helpers/typographyVariables');
+import * as dotenv from 'dotenv';
+import {
+	Dict,
+	ITokenJsonObject,
+	IResultJsonObject,
+	ConfigFileObject,
+	ITypographyToken,
+	ConfigFileBin,
+} from './helpers/structures';
+
+dotenv.config();
+
+const ImportTypographyRaw = (
+	name: string,
+	binValues: ConfigFileObject,
+	jsonTypography: Object,
+	isTheme: boolean,
+) => {
+	let resultCss: Array<string> = [];
+	let resultScss: Array<string> = [];
+	let resultJsonCss: Dict<Array<IResultJsonObject>> = {};
+
+	resultCss.push(modifiers.cssFileFirstLine(isTheme, name));
```

Comment:
we defined a `name` in config, we passed it through many layers and finally... It's ignored inside `cssFileFirstLine()`. Can we simplify it?

Reply:
Fixed.